content/docs/user-guide/how-to/best-practices.md

```diff
@@ -0,0 +1,136 @@
+  changes contributed the most in improving target
+  [metric](doc/command-reference/metrics). You can monitor the degree of each
+  change.
+
+- Navigating through experiments
+
+  To recover a model from last week without wasting time required for the model
+  to retrain, first you can checkout the required `revision`. Followed by
+  `dvc checkout` to update DVC-tracked files and directories in your workspace.
+
+- Switching between datasets
+
+  You can quickly switch between a large dataset and a small subset without
+  modifying source code. To achieve this yoe need to change dependencies of
+  relevant stage either by using `dvc run` with `-f` option or by manually
+  editing the stage in `dvc.yaml` file.
+
+- Reproducibility
+
+  You can run a model's evaluation process again without actually retraining the
+  model and preprocessing a raw dataset. DVC provides a way to reproduce
+  pipelines partially. You can use `dvc repro` to execute evaluation stage
+  without reproducing complete pipeline:
+
+  ```dvc
+  $ dvc repro evaluate
+  ```
+
+- Managing and sharing large data files
+
+  Cloud or local storage can be used to store the project's data. You can share
+  the entire 147 GB of your ML project, with all of its data sources,
+  intermediate data files, and models with others if they are stored on
+  [remote storage](doc/command-reference/remote/add#supported-storage-types).
+  Using this you can share models trained in a GPU environment with colleagues
+  who don't have access to a GPU. Have a look at this
+  [example](doc/command-reference/pull#example-download-from-specific-remote-storage)
+  to see how this works.
+
+- Manually editing dvc.yaml or .dvc files
+
+  It's safe to edit `dvc.yaml` and `.dvc` files. You can manually change all the
+  fields present in these files. However, please keep in mind to not change the
+  `md5` or `checksum` fields in `.dvc` files as they contain hash values which
+  DVC uses to track the file or directory.
+
+- Never store credentials in project config
+
+  Do not store any user credentials in project config file. This file can be
+  found by default in `.dvc/config`. Use `--local`, `--global`, or `--system`
+  command options with `dvc config` for storing sensitive, or user-specific
+  settings:
+
+  ```dvc
+  $ dvc config --system remote.username [password]
+  ```
+
+- Tracking <abbr>outputs</abbr> by Git
+
+  If `outs` are small files in size and you want to track them with Git then you
+  can use `--outs-no-cache` option to define outputs while creating or modifying
+  a stage. DVC will not track will not track outputs in this case:
+
+  ```dvc
+  $ dvc run -n train -d src/train.py -d data/features \
+            ---outs-no-cache model.p \
+            python src/train.py data/features model.pkl
+  ```
+
+---
+
+## Questions on...
+
+### Source code and data versioning
+
+- How do you avoid discrepancies between
```

Comment:
Looks like everything under here is repeated now. It's fine to remove it if it has been rewritten in a how-to format.